src/common/utils/messageHandlers.ts

```diff
@@ -1,42 +1,19 @@
-import { InternalMethods } from '@common/types/message-types';
 import { BackgroundMessages } from 'common/types/messages';
 
 function validateMessagesAreFromExtension(sender: chrome.runtime.MessageSender) {
   // Only respond to internal messages from our UI, not content scripts in other applications
   return sender.url?.startsWith(chrome.runtime.getURL(''));
 }
 
-let inMemoryKey = '';
-
 async function internalBackgroundMessageHandler(
   message: BackgroundMessages,
   sender: chrome.runtime.MessageSender,
   sendResponse: (response?: any) => void,
 ) {
   if (!validateMessagesAreFromExtension(sender)) {
+    sendResponse();
```

Comment:
why did this logic change?

it looks like the if block is redundant now

Reply:
we were utilizing the background worker for storing the `seedphrase` in memory we don't do that anymore with the `SeedVault` , sendResponse is only there to indicate that the handler was terminated
 ```
 export enum InternalMethods {
  RequestDerivedStxAccounts = 'RequestDerivedStxAccounts',
  ShareInMemoryKeyToBackground = 'ShareInMemoryKeyToBackground',
  RequestInMemoryKeys = 'RequestInMemoryKeys',
  RemoveInMemoryKeys = 'RemoveInMemoryKeys',
  OriginatingTabClosed = 'OriginatingTabClosed',
}
```